perllib/FixMyStreet/Cobrand/Merton.pm

```diff
@@ -76,6 +76,19 @@ sub open311_extra_data_include {
     return [];
 }
 
+sub open311_get_update_munging {
+    my ($self, $comment) = @_;
+
+    my $latest = $comment->problem->comments->search(undef, {
```

Comment:
Does it make sense to search the most recent visible comment rather than all? Can't think why it would matter but the question popped into my head.

```suggestion
    my $latest = $comment->problem->comments->search({ state => 'confirmed' }, {
```

Reply:
Yeah, makes sense.